include/npcomp/Conversion/CMakeLists.txt

```diff
@@ -2,4 +2,4 @@ set(LLVM_TARGET_DEFINITIONS Passes.td)
 mlir_tablegen(Passes.h.inc -gen-pass-decls ${NPCOMP_TABLEGEN_ARGS})
 add_public_tablegen_target(NPCOMPConversionPassIncGen)
 
-add_mlir_doc(Passes -gen-pass-doc ConversionPasses ./)
+add_mlir_doc(Passes -gen-pass-doc NPCOMPConversionPasses ./)
```

Comment:
This was conflicting with the MLIR ConversionPasses name.

Reply:
Thx. Copy-pasta from the beginning of the project.